src/components/modal/ModalDelete.vue

```diff
@@ -0,0 +1,57 @@
+<template>
+  <the-modal>
+    <div class="modal__delete">
+      <p>Are you sure want to delete?</p>
+      <div>
+        <base-button
+          type="submit"
```

Comment:
This button is not in a form, having it with a submit type might cause some interesting behaviour depending on circumstances.
```suggestion
```